template/tests/test_extension.py.jinja

```diff
@@ -2,7 +2,7 @@ from mopidy_{{ ext_name }} import Extension
 from mopidy_{{ ext_name }} import frontend as frontend_lib
 
 
-def test_get_default_config():
+def test_get_default_config() -> None:
```

Comment:
Technically correct, but this doesn't really add any value? I can see the point in annotating any fixtures though

Reply:
In existing extensions, we've usually disabled ruff's `ANN` rules for `tests/*`. I don't really have any problems with going all in on type checking in tests too, especially when done from the start of a project, so I decided to leave `ANN` enabled for now, and thus wanted the initially generated test file to pass ruff out of the box.